src/main/scala/com/advancedtelematic/director/db/AssignmentCancelation.scala

```diff
@@ -0,0 +1,28 @@
+package com.advancedtelematic.director.db
+
+import com.advancedtelematic.director.data.DbDataType.Assignment
+import com.advancedtelematic.libats.data.DataType.Namespace
+import com.advancedtelematic.libats.messaging_datatype.DataType.DeviceId
+import slick.jdbc.MySQLProfile.api._
+import com.advancedtelematic.libats.slick.db.SlickAnyVal._
+import com.advancedtelematic.libats.slick.db.SlickUUIDKey._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object AssignmentCancelation {
```

Comment:
hm. I wonder if would be best to have all this in the same `Repository` and keep `DBIO` handling inside the `repository` package? Then an `AssignmentRepository` would handle both processed and current assignments. Handling two tables would just be an internal detail to that class.

Reply:
You mean keeping DBIO in the db package?

Reply:
Makes sense.